packages/builtin_test/_test.pony

```diff
@@ -1092,6 +1093,15 @@ class iso _TestStringChop is UnitTest
     h.assert_eq[String box](expected_left, consume left)
     h.assert_eq[String box](expected_right, consume right)
 
+class iso _TestStringConcatOffsetLen is UnitTest
+  """
+  Test String.concat working correctly for non-default offset and len arguments
+  """
+  fun name(): String => "builtin/String.concat"
+
+  fun apply(h: TestHelper) =>
+    h.assert_eq[String](recover String .> concat("ABCD".values(), 1, 2) end, "BC")
```

Comment:
to match with standard library programming style, this should be `String.>concat`